pytest/tests/loadtest/locust/common/ft.py

```diff
@@ -75,3 +124,55 @@ def sign_and_serialize(self, block_hash):
                                                  int(3E14), int(1E23),
                                                  account.use_nonce(),
                                                  block_hash)
+
+
+@events.init.add_listener
+def on_locust_init(environment, **kwargs):
+    # Note: These setup requests are not tracked by locust because we use our own http session
+    host, port = environment.host.split(":")
+    node = cluster.RpcNode(host, port)
+
+    ft_contract_code = environment.parsed_options.fungible_token_wasm
+    num_ft_contracts = environment.parsed_options.num_ft_contracts
+    funding_account = NearUser.funding_account
+    parent_id = funding_account.key.account_id
+    worker_id = "local_id"
+    if hasattr(environment.runner, "worker_id"):
+        # we are running a multi-node setup
+        worker_id = environment.runner.worker_id
+
+    funding_account.refresh_nonce(node)
+
+    environment.ft_contracts = []
+    # TODO: Create accounts in parallel
+    for i in range(num_ft_contracts):
+        # Prefix that makes accounts unique across workers
+        prefix = str(hash(str(worker_id) + str(i)))[-6:]
```

Comment:
Why do we need to take a hash here and can't just use a simpler `f"{worker_id}_{i}"` which also should be unique?

Reply:
Shuffling with a hash avoids locality in the state trie.

I will add this as a comment.

Also, we should be more careful with the hash, since this always produced a number and we want to have all valid account id characters in the output in order to cover all shards. The same is (even more) true for account generation in `NearUser`. I'll add TODOs to the code but don't want to fix it as part of this PR.